db/migrations/20170915150351_drop_title_from_movies.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.schema.table('movies', (table) => {
+    table.dropColumn('title');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ADD CONSTRAINT movies_name_not_null CHECK (name IS NOT NULL) NOT VALID')
+    .then(() => Knex.raw('ALTER TABLE movies VALIDATE CONSTRAINT movies_name_not_null'));
```

Comment:
nit: we try an avoid nesting promises like this when not necessary.
```js
return Knex.schema.table( ... )
.then(() => Knex.raw( ... )) // Add Constraint
.then(() => Knex.raw( ... )); // Validate Constraint
```

Reply:
I think I merged simultaneously when this was posted. I've made a mental note in the future to flatten my promise chains, when possible.